### PR TITLE
Add stats for super admin

### PR DESCRIPTION
**Pull request by georgeschena**
screenshot 2019-02-26 at 15 29 00

### CODE REVIEW

File: app/controllers/admin/organisations_controller.rb [hunk 1]
```diff
@@ -3,6 +3,10 @@ class Admin::OrganisationsController < AdminController
 
   def index
     @organisations = Organisation.includes(:signed_mou_attachment).order("#{sort_column} #{sort_direction}").all
+
+    @total_number_of_organisations = Organisation.all.count
+    @total_number_of_locations = Location.all.count
+    @total_number_of_ips = Ip.all.count
```
Comment (tommotaylor) [inline app/controllers/admin/organisations_controller.rb]: `Location.count` and `Ip.count` is more succinct.

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -29,3 +29,12 @@ $govuk-font-url-function: "font-url";
...
   vertical-align: middle;
 }
+
+.total-org-stat-results-title {
+  font-size: 63px;
+  padding-right: 54px;
+}
+
+.admin-panel-stats-container {
+  padding-left: 125px;
```
Comment (tommotaylor) [inline app/assets/stylesheets/application.scss]: This seems a bit weird. Are you using padding to centre the container? If so, you should use something like flexbox instead.

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -12,6 +12,7 @@ $govuk-font-url-function: "font-url";
 @import "./components/sub_nav";
 @import "./components/team_members";
 @import "./utilities/typography";
+@import "./components/admin_stats_panel.scss";
```
Comment (tommotaylor) [inline app/assets/stylesheets/application.scss]: This list used to be alphabetical. 😞

File: app/views/admin/organisations/index.html.erb [hunk 4]
```diff
@@ -1,7 +1,31 @@
-<h1 class="govuk-heading-l">All organisations</h1>
-<h3 class="govuk-heading-m">
-  <%= pluralize(@organisations.count, 'organisation') %> are signed up to GovWifi
-</h3>
+<h1 class="govuk-heading-l">Summary of all organisations</h1>
+<div class="govuk-grid-column admin-panel-stats-container">
+  <div class="govuk-grid-column-one-third stats-align-center">
+    <div class="govuk-heading-l">
+    <strong class="total-org-stat-results-title"><%= @total_number_of_locations%></strong>
```
Comment (emileswarts) [inline app/views/admin/organisations/index.html.erb]: Closing tag whitespace is off, not sure why rubocop isn't picking up on this.
Have added to tech debt board to check

**Comment by kjdchapman** [comment]
Had a conversation with Steve, and think we can do something much simpler here. 

He's certainly interested in how many locations we have total (he gets asked that fairly regularly!), but not interested in how many IP addresses we have total. 

Going to put up another PR to demonstrate.

**Comment by kjdchapman** [comment]
Suggested an alternate approach in https://github.com/alphagov/govwifi-admin/pull/503

**Pull request closed without merging**